### ISSUE TITLE
Can't install wazimap

### ISSUE THREAD

**Issue by matiasf** [Code4SA/wazimap#51]
Hello guys and gals,

I have forked recently this repository because we want to try adapt wazimap for the Uruguay census.
We are trying to install wazimap, I tried on my machine and in a vagrant box, both without success. 

This is part of the error.

`==> default:       File "/tmp/pip-build-FFEvhW/GDAL/setup.py", line 196, in finalize_options
==> default:         self.gdaldir = self.get_gdal_config('prefix')
==> default:       File "/tmp/pip-build-FFEvhW/GDAL/setup.py", line 176, in get_gdal_config
==> default:         return fetch_config(option)
==> default:       File "/tmp/pip-build-FFEvhW/GDAL/setup.py", line 129, in fetch_config
==> default:         raise gdal_config_error, e""")
==> default:       File "<string>", line 4, in <module>
==> default:     __main__.gdal_config_error: [Errno 2] No such file or directory
`

This is a gist with the Vagrant installation I have create to test this.
https://gist.github.com/matiasf/ef04084d76ea084fe4bd9a359376c990

Some of you know which can be the problem.

Regards,
Matías.

**Comment by longhotsummer** [comment]
Hi Matías,

It looks like gdal isn't installed. Try installing with `apt-get install gdal-bin`. Could you verify the version with:

```
$ ogrinfo --version
GDAL 1.11.1, released 2014/09/24
```

If that's the cause, then we can update the docs to make that clearer.

**Comment by matiasf** [comment]
Hi Greg,

I have do that but continue with the same error. I send you the version of the ogrinfo.

`vagrant@vagrant-ubuntu-trusty-64:~$ ogrinfo --version`
`GDAL 1.10.1, released 2013/08/26`

This is the vagrant with the gdal-bin install.

https://gist.github.com/matiasf/ef04084d76ea084fe4bd9a359376c990

Thanks for the quick answer.

Regards,
Matías.

**Comment by longhotsummer** [comment]
This turned out to be a bit complicated. I have released version 0.4.6 of Wazimap which makes it easier.

In particular:
- Allows GDAL >=1.11.0 and < 2.0.
- Lazy load GDAL in case a user wants to bypass that (we might make this an optional dependency later, since it's so complicated)
- Install django < 1.10

You'll need to add the equivalent of the following to your vagrant file:

```
sudo add-apt-repository ppa:ubuntugis/ppa
sudo apt-get update
sudo apt-get install binutils libproj-dev gdal-bin python-gdal libpq-dev python-dev
```

Please also change `pip install django` to `pip install 'django<1.10'`.

Does that get Wazimap up and running?

**Comment by matiasf** [comment]
Hi Greg,

It work! Thanks a lot, i'm going to continue withe configuration stuff. If I have some other trouble you are going to see me here again.

Thanks a lot,
Matías.
